OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -732,6 +740,10 @@ public void SetPosition(Actor self, WPos pos)
 			else if (!isCruising && cruising)
 				OnCruisingAltitudeLeft();
 
+			if (notify && notifyVisualPositionChanged != null)
```

Comment:
We might want to have a comment here like `This might be called from the ctor before Created ran`? Just throwing an idea out here: We could also only set `notify` to true in `Created` instead of initialising it with `true`.

Reply:
Added a comment. 

I consider the `notify` flag a bit of a hack, which can hopefully go away when we get around to revising the aircraft repulsion logic, so I'd rather not make anything else reliant on it.